src/linux/linux_io.c

```diff
@@ -335,6 +351,50 @@ static int create_server_socket_all_interfaces(int port)
 	return -1;
 }
 
+static int create_server_unix_domain_socket(const char *pPath)
+{
+	int listen_fd = socket(AF_UNIX, SOCK_STREAM, 0);
+	if (unlikely(listen_fd < 0)) {
+		log_err("Could not create listen socket!\n");
+		return -1;
+	}
+
+	static const int reuse_on = 1;
+	if (unlikely(setsockopt(listen_fd, SOL_SOCKET, SO_REUSEADDR, &reuse_on,
```

Comment:
Why do we need SO_REUSEADDR for and UDS?

Reply:
If I remove it the second client won't connect...